tensorflow/core/platform/cpu_feature_guard.cc

```diff
@@ -138,8 +138,16 @@ void InfoAboutUnusedCPUFeatures() {
     CheckIfFeatureUnused(CPUFeature::FMA, "FMA", missing_instructions);
 #endif  // __FMA__
 #endif  // else of if defined(_MSC_VER) && !defined(__clang__)
-    if (!missing_instructions.empty()) {
-      LOG(INFO) << "This TensorFlow binary is optimized with Intel(R) MKL-DNN "
+
+    string intel_library_official_name(
+        "Intel(R) oneAPI Deep Neural Network Library (oneDNN) ");
```

Comment:
I'm not sure I understand why we have Intel(R) here

Reply:
If it's not required (since this is all open source), I'd like to removed it and make two strings the same.

@nammbash Can we use one string (without `Intel(R)`) for both?

Reply:
@penpornk have to check internally. Will revert soon.

Reply:
@penpornk  @vpirogov 
the string is the same now. Need Re-review